### PR TITLE
V1.3.0 release

### PR DESCRIPTION
**Pull request by opotowsky**
Probably for @gidden or maybe @scopatz.
I'd like to get CI for PRs working while not understanding how this works. Here is a start using what @gidden wrote as a template. Is there anything else I need to do?

### CODE REVIEW

**Comment by scopatz** [comment]
> I'd like to get CI for PRs working while not understanding how this works.

Start by reading the Travis docs.

**Comment by gonuke** [comment]
It is probably also very valuable to look at the cycamore travis files

**Comment by gonuke** [comment]
You'll also need a set of files in a directory called `conda-recipes` with models for these in cycamore also

**Comment by opotowsky** [comment]
Thanks @gonuke! I added them yesterday (https://github.com/opotowsky/cymetric/tree/v1.3.0-release) but am trying to resolve this failure (https://travis-ci.org/opotowsky/cymetric) on a different branch from @gidden

File: .travis-install.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+set -x # print cmds
+set -e # exit as soon as an error occurs
+
+# log
+msg=`git log --pretty=oneline -1`
+echo "Building commit: $msg" 
+
+# setup conda recipe to use develop cyclus
+sed -i  "s/- cyclus/- cyclus 0.0/g" conda-recipe/meta.yaml
```
Comment (gidden) [inline .travis-install.sh]: @opotowsky, try removing this line

**Comment by opotowsky** [comment]
ping @scopatz 

We had a discussion on this internal PR (https://github.com/opotowsky/cymetric/pull/2) about the failure of travis CI (https://travis-ci.org/opotowsky/cymetric/jobs/61680159#L1911). glib is included in meta.yaml for cymetric, as this file was copied directly from the ciclus repo (https://github.com/opotowsky/cymetric/blob/v1.3.0-release/conda-recipe/meta.yaml#L27). @gidden and I are unable to discern the source of this error; do you have any insight?

**Comment by opotowsky** [comment]
rebased to only include travis commits

**Comment by scopatz** [comment]
Can you please revert these changes and open a new PR @Opotowsky? Merging your own PR is not something we allow. Thanks.

**Comment by opotowsky** [comment]
I am fully aware of what we allow and just made a silly mistake. It is now reverted.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**